### DISCUSSION TITLE
is this normal?

### DISCUSSION THREAD

**Discussion by ghost** [VKCOM/VKUI#3964] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/73018753/211650882-b3309b2c-b884-4e3c-9b70-4b6a69f4eefd.png)
i have black background after placeholder.
placeholder is gray,
is this normal?!

(i am russian, sorry for my english)

**Comment by inomdzhon** [comment, marked as answer]
Привет, @astrofox09 

Какую версию VKUI ты используешь? Если **v5**, то ниже опишу возможную причину проблемы и её решение.

## Причина

### v4

В **v4** `AppRoot` программно добавлял CSS селектор `vkui__root` для **root-элемента** приложения:

https://github.com/VKCOM/VKUI/blob/c1a19e5462dc9ecc15da58d5cf4249c0efc76b6e/src/components/AppRoot/AppRoot.tsx#L120-L123

А для `<AppRoot mode="partial">` необходимо было добавлять этот селектор вручную ([ссылка на документацию](https://vkcom.github.io/VKUI/4.40.0/#/Modes)).

На этот CSS селектор применялось св-во `height: 100%`:

https://github.com/VKCOM/VKUI/blob/c1a19e5462dc9ecc15da58d5cf4249c0efc76b6e/src/styles/common.css#L5-L11

### v5

В **v5** мы убрали необходимость в селекторе `vkui__root`. Соотвественно, из CSS правила тоже убрали этот селектор:

https://github.com/VKCOM/VKUI/blob/d737403a386ffed9007e33fb60e094be2b6bdb8c/src/styles/common.css#L5-L10

Это привело к тому, что теперь `height: 100%` надо добавлять вручную на свой **root-элемент**.

## Решение

Попробуй, пожалуйста, установить `height: 100%` для своего **root-элемента**.

**Reply by ghost** [reply]
использую v5, исправил, всё работает, огромное спасибо!!
